packages/nuxt/src/components/plugins/tree-shake.ts

```
@@ -197,62 +191,61 @@
+    }
+  } else if (componentCall.type === 'CallExpression') {
+    return getComponentName(componentCall)
   }
-  return (componentCall.arguments[0] as Identifier).name
 }
 
 /**
  * remove a variable declaration within the code
  */
-function removeVariableDeclarator (codeAst: Node, name: string, magicString: MagicString, removedNodes: WeakSet<Node>): AcornNode<Node> | void {
+function removeVariableDeclarator (codeAst: Program, name: string, magicString: MagicString, removedNodes: WeakSet<Node>): Node | void {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)